test/e2e/commit_test.go

```diff
@@ -22,6 +22,14 @@ var _ = Describe("Podman commit", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 
+		if !IsRemote() {
+			messages := session.ErrorToString()
+			Expect(messages).To(ContainSubstring("Getting image source signatures"))
+			Expect(messages).To(ContainSubstring("Copying blob"))
+			Expect(messages).To(ContainSubstring("Writing manifest to image destination"))
+			Expect(messages).To(Not(ContainSubstring("level=")), "Unexpected logrus messages in stderr")
+		}
```

Comment:
Are you sure there are not logged on podman-remote?

```
$ podman-remote pull alpine
Trying to pull docker.io/library/alpine:latest...
Getting image source signatures
Copying blob sha256:7264a8db6415046d36d16ba98b79778e18accee6ffa71850405994cffa9be7de
Copying config sha256:7e01a0d0a1dcd9e539f8e9bbd80106d59efbdf97293b3d38f5d7a34501526cdb
Writing manifest to image destination
7e01a0d0a1dcd9e539f8e9bbd80106d59efbdf97293b3d38f5d7a34501526cdb
```

Reply:
Extremely sure. Try **commit** instead of `pull`. You will see the messages on the server's stderr, but nothing on the client.

Reply:
Weird that we get this output across for pull but not commit. Oh well just another sad story for the remote client logging.

Reply:
Filed #19947